casual/casual-network/src/main/java/se/laz/casual/network/inbound/reverse/ReverseInboundConnectionInformation.java

```diff
@@ -33,6 +33,7 @@ public class ReverseInboundConnectionInformation
     private final String domainName;
     private final Class<? extends Channel> channelClass;
     private boolean useEpoll;
+    private final long maxConnectionBackoffMillis;
```

Comment:
Now we are in the context of a `ReverseInboundConnectionInformation`, do we need to repeat the `Connection` in the variable? `maxBackoffMillis` for readability/brevity perhaps?

Reply:
Yeah, your are probably right. That's what I had at first and felt like it might be unclear, but it really shouldn't be in the context of the class it's in. I'll fix it